websock/http/server.nim

```diff
@@ -207,18 +206,34 @@ proc accept*(server: HttpServer): Future[HttpRequest]
 
 proc create*(
   _: typedesc[HttpServer],
-  address: TransportAddress,
+  address: TransportAddress | string,
   handler: HttpAsyncCallback = nil,
   flags: set[ServerFlags] = {},
-  handshakeTimeout = HttpHeadersTimeout): HttpServer
+  headersTimeout = HttpHeadersTimeout,
+  handshakeTimeout = 0.seconds
+  ): HttpServer
   {.raises: [Defect, CatchableError].} = # TODO: remove CatchableError
   ## Make a new HTTP Server
   ##
 
-  var server = HttpServer(handler: handler, handshakeTimeout: handshakeTimeout)
+  var server = HttpServer(
+    handler: handler,
+    headersTimeout: headersTimeout,
+    handshakeTimeout:
+      if handshakeTimeout == 0.seconds:
+        headersTimeout + headersTimeout div 20
```

Comment:
I still don't get why we need to divide by 20.

Reply:
It's just a sane default, handshakeTimeout = headersTimeout * 1.05
The div 20 is just to stay in integer world, which is generally better

Having headersTimeout * 2 is too big imo, that basically means we wait for headersTimeout to read and headers, and then leave headersTimeout again to send the error. Too long, imo

Reply:
OK, I'm not opposed to the argument, I guess I'm missing where does the `1.05` comes from - are you talking about operator precedence `headersTimeout + (headersTimeout div 20)`? I guess I'm missing something here.

Reply:
```suggestion
        # default to headersTimeout * 1.05
        headersTimeout + (headersTimeout div 20)
```